trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesExperimentService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Tripal Cultivate Phenotypes Experiments/Projects service definition.
+ */
+
+namespace Drupal\trpcultivate_phenotypes\Service;
+
+/**
+ * Class TripalCultivatePhenotypesExperimentService.
+ */
+class TripalCultivatePhenotypesExperimentService {
```

Comment:
Why do we need this service? Shouldn't experiments be entered using the project autocomplete you made for Tripal core? Please either remove this or actually define what services will be used for in the file docblock.